src/gurt/telemetry.c

```diff
@@ -2411,7 +2413,7 @@ d_tm_get_shared_memory(int srv_idx)
 }
 
 /**
- * Allocates memory from within the shared memory pool with 16-bit alignment
+ * Allocates memory from within the shared memory pool with 64-bit alignment
```

Comment:
That will work to go to 64-bit alignment, but I don't think it is necessary.  Did you find a situation that caused an alignment error?  Or was there a performance reason you encountered?

Reply:
Yes, I kept hitting this runtime error until I changed the alignment: https://sourcegraph.com/github.com/golang/go@4a70ff425b3c16c19785b04bb89ca856749ed65b/-/blob/src/runtime/checkptr.go#L18